ui/components/component-library/avatar-favicon/avatar-favicon.js

```diff
@@ -28,17 +29,21 @@ export const AvatarFavicon = ({
       display={DISPLAY.FLEX}
       alignItems={ALIGN_ITEMS.CENTER}
       justifyContent={JUSTIFY_CONTENT.CENTER}
-      className={classnames('avatar-favicon', className)}
+      className={classnames('mm-avatar-favicon', className)}
       {...{ borderColor, ...props }}
     >
-      {imageSource ? (
-        <img className="avatar-favicon__image" src={imageSource} alt={imgAlt} />
+      {src ? (
+        <img
+          className="mm-avatar-favicon__image"
+          src={src}
+          alt={`${name} logo` || 'dapp logo'}
+        />
       ) : (
         <Icon
           name={ICON_NAMES.GLOBAL_FILLED}
           color={COLORS.ICON_DEFAULT}
           size={size}
-          aria-label={imgAlt}
+          aria-label={name}
```

Comment:
Hmm I'm trying to think if this requires an aria-label... It kind of depends on the use case for this whole component. Sometimes it could represent that an account is connected when it is displayed by itself in a navbar for instance. In other cases it will be as apart of the TagUrl component in that case it's just used decoratively. This would be a good use case for an accessibility section. In the meantime I think we can just remove the aria-label here.

Reply:
Should also fix this error 

<img width="788" alt="Screenshot 2022-12-20 at 5 44 42 PM" src="https://user-images.githubusercontent.com/8112138/208800949-6e62896a-040b-414c-8def-53a8506e87d2.png">